app/layout.tsx

```diff
@@ -5,7 +5,7 @@ import { Header } from "@/components/customs/header";
 import React from "react";
 import clsx from "clsx";
 import "mapbox-gl/dist/mapbox-gl.css";
-
+import { SpeedInsights } from "@vercel/speed-insights/next"
```

Comment:
Will just give us some user stats in the vercel dashbaord. not sure how useful but thought we could test it out